web-client/src/views/Public/PublicSearch.tsx

```diff
@@ -41,52 +129,70 @@ export const PublicSearch = connect(
           <SuccessNotification />
           <WarningNotification />
 
-          <Tabs
-            bind="advancedSearchTab"
-            className="classic-horizontal-header3 tab-border"
-            headingLevel="2"
-            onSelect={() => {
-              advancedSearchTabChangeSequence();
-            }}
-          >
-            <Tab id="tab-case" tabName="case" title="Case">
-              <SearchBoilerplateText formTypeText="a case" />
-              <CaseSearchForm
-                submitAdvancedSearchSequence={
-                  submitPublicCaseAdvancedSearchSequence
-                }
-                submitDocketNumberSearchSequence={
-                  submitPublicCaseDocketNumberSearchSequence
-                }
-              />
-              <SearchResults />
-            </Tab>
-            <Tab data-testid="order-search-tab" tabName="order" title={'Order'}>
-              <SearchBoilerplateText formTypeText="an order" />
-              <OrderSearchForm
-                submitAdvancedSearchSequence={
-                  submitPublicOrderAdvancedSearchSequence
-                }
-              />
-              <DocumentSearchResults />
-            </Tab>
-            <Tab
-              data-testid="opinion-search-tab"
-              tabName="opinion"
-              title={'Opinion'}
+          <NonMobile>
+            <Tabs
+              bind="advancedSearchTab"
+              className="classic-horizontal-header3 tab-border"
+              headingLevel="2"
+              onSelect={() => {
+                advancedSearchTabChangeSequence();
+              }}
             >
-              <SearchBoilerplateText
-                formTypeText="an opinion"
-                isOpinion="true"
-              />
-              <OpinionSearchForm
-                submitAdvancedSearchSequence={
-                  submitPublicOpinionAdvancedSearchSequence
-                }
-              />
-              <DocumentSearchResults />
-            </Tab>
-          </Tabs>
+              <Tab id="tab-case" tabName="case" title="Case">
+                {CaseRender()}
+              </Tab>
+              <Tab
+                data-testid="order-search-tab"
+                tabName="order"
+                title={'Order'}
+              >
+                {OrderRender()}
+              </Tab>
+              <Tab
+                data-testid="opinion-search-tab"
+                tabName="opinion"
+                title={'Opinion'}
+              >
+                {OpinionRender()}
+                <DocumentSearchResults />
```

Comment:
Is this accidentally rendered twice? I see it's also called in the `OpinionRender` function